hack/generate-bindata.sh

```diff
@@ -65,7 +65,7 @@ fi
 rm -f "${BINDATA_OUTPUT}.tmp"
 
 # These are files for runtime code
-BINDATA_OUTPUT="pkg/kubectl/generated/bindata.go"
+BINDATA_OUTPUT="staging/src/k8s.io/kubectl/pkg/generated/bindata.go"
```

Comment:
I assume this will eventually move to something run in k8s.io/kubectl?

Reply:
exactly